dangerzone/cli.py

```diff
@@ -90,7 +92,7 @@ def cli_main(
             exit(1)
 
     # Ensure container is installed
-    global_common.install_container()
+    container.install()
```

Comment:
Check return value to ensure that the installation is successful.

Reply:
I see that I didn't do that either, so maybe I'm just forgetting something here. Up to you!

Reply:
That may also just be outside the scope of this PR.

Reply:
Oh, never mind all of this! You added that "container-tech-checking logic" elsewhere. In that case, I'd just change the comment now to say "install container".